README.md

```diff
@@ -17,7 +17,16 @@ $ bpanel-cli install @bpanel/bwallet
 This will update the `bpanel` config file so that the plugins array contains `@bpanel/bwallet`.
 `bpanel` will install the plugin during its build.
 
+Note: To create a self signed certificate to run locally, use the command:
```

Comment:
Maybe an extra "which is necessary for running over https and interacting with a multisig wallet server". 

We should also probably include in bpanel readme, specifically in configuration where we already mention this briefly: https://github.com/bpanel-org/bpanel#configuration